### DISCUSSION TITLE
Ultralytics Live Session 4: Accelerating Model Deployment With Neural Magic

### DISCUSSION THREAD

**Discussion by taliabender** [ultralytics/ultralytics#674] [Show and tell]
When we polled the community, we found that 22% percent of you experienced difficulty deploying your vision AI models. To improve this step in the ML pipeline, we've partnered with [Neural Magic](https://www.linkedin.com/company/neural-magic/), whose DeepSparse tool takes advantage of sparsity and low-precision arithmetic within neural networks to offer exceptional performance on commodity hardware. 🚀

In our next Ultralytics Live Session on February 8th, [Michael Goin](https://www.linkedin.com/in/ACoAAAyLJPsBKgTOxZF4WlLN_Qko3KNhUF13CNM) will join [Glenn Jocher](https://www.linkedin.com/in/ACoAAASa684BOUpWeSgSJbXKXU7MaWJqS4RCFes) to discuss how to achieve GPU-class performance for YOLOv5 on commodity CPUs.

Be sure to [add the event to your calendar](https://www.linkedin.com/events/ultralyticslivesessions4-accele7024665296598634496/) and come prepared with your questions! 🔥

![LI:TW:YT Ultralytics Live Sessions](https://user-images.githubusercontent.com/85292283/215147923-186e943d-209d-41b1-a2e5-09fe86f0e77f.png)

**Comment by glenn-jocher** [comment]
@taliabender thank you for sharing the excitement about our upcoming Ultralytics Live Session! It's great to see the community's interest in optimizing model deployment. While the session you mentioned focuses on YOLOv5, I'd like to remind everyone that similar principles and tools can also be applied to YOLOv8.

For those working with YOLOv8 and interested in deployment acceleration, I encourage you to explore our documentation at https://docs.ultralytics.com. Here, you'll find comprehensive guides on how to train, validate, and deploy YOLOv8 models efficiently.

Remember, whether you're aiming for real-time object tracking or deploying on edge devices, YOLOv8 is designed to deliver high performance. Stay tuned for more updates and feel free to reach out with any questions regarding YOLOv8. Happy coding! 🌟
